src/DssSpell.sol

```diff
@@ -32,6 +51,36 @@ contract DssSpellAction is DssAction {
         return false;
     }
 
+    address immutable internal OPTIMISM_GOV_RELAY = DssExecLib.getChangelogAddress("OPTIMISM_GOV_RELAY");
+    address immutable internal ARBITRUM_GOV_RELAY = DssExecLib.getChangelogAddress("ARBITRUM_GOV_RELAY");
+    address immutable internal STARKNET_GOV_RELAY = DssExecLib.getChangelogAddress("STARKNET_GOV_RELAY");
+
+    address immutable internal DAI = DssExecLib.getChangelogAddress("MCD_DAI");
+    address immutable internal STARKNET_ESCROW = DssExecLib.getChangelogAddress("STARKNET_ESCROW");
+    address immutable internal STARKNET_DAI_BRIDGE_LEGACY = DssExecLib.getChangelogAddress("STARKNET_DAI_BRIDGE_LEGACY");
+
+    address constant internal OPTIMISM_L2_SPELL = 0xC077Eb64285b40C86B40769e99Eb1E61d682a6B4;
```

Comment:
Spell NOT verified at https://blockscout.com/optimism/goerli/address/0xC077Eb64285b40C86B40769e99Eb1E61d682a6B4

Is there a better explorer?